show_client/mac_aging_time_cli.go

```diff
@@ -0,0 +1,48 @@
+package show_client
```

Comment:
Perhaps we can declare one mac_cli.go file for the other mac commands to be supported.